src/test/unit_tests/test_s3_ops.js

```diff
@@ -37,14 +37,15 @@ const text_file5 = 'text-file5';
 
 mocha.describe('s3_ops', function() {
 
-    let s3;
-
+    let s3_local;
+    let REMOTE_SOURCE_BUCKET;
+    let REMOTE_TARGET_BUCKET;
     mocha.before(async function() {
         const self = this; // eslint-disable-line no-invalid-this
         self.timeout(60000);
 
         const account_info = await rpc_client.account.read_account({ email: EMAIL, });
-        s3 = new AWS.S3({
+        s3_local = new AWS.S3({
```

Comment:
What is the purpose of this rename from s3 to s3_local?
Can we revert it?

Reply:
Reverted.